socialhome/content/serializers.py

```diff
@@ -91,3 +94,6 @@ def get_user_has_shared(self, obj):
         if not request:
             return False
         return Content.has_shared(obj.id, request.user.profile.id) if hasattr(request.user, "profile") else False
+
+    def get_tags(self, obj):
```

Comment:
While this works, it's inefficient. REST framework has a thing to do this, it's the [SlugRelatedField](http://www.django-rest-framework.org/api-guide/relations/#slugrelatedfield). Using that exactly as the example there (with `read_only` and `many`, field name `name`) should work nicely and be well optimized.